components/ai_chat/core/browser/engine/conversation_api_client_unittest.cc

```diff
@@ -308,6 +352,21 @@ TEST_F(ConversationAPIUnitTest, PerformRequest_PremiumHeaders) {
         EXPECT_EQ(queries[0], "Star Wars");
         EXPECT_EQ(queries[1], "Star Trek");
       });
+  EXPECT_CALL(mock_callbacks, OnDataReceived(_))
```

Comment:
what does it look like when something in here fails? While this may be convenient, I'm a bit concerned that failures will be very hard to understand and trace back